Module-6-Assignment/ClassOperationsTriggerTest.cls

```diff
@@ -0,0 +1,74 @@
+/**
+* Test Class for testing ClassOperationsTrigger.
+*/
+@isTest
+public class ClassOperationsTriggerTest {
+    
+    /**
+	  * Test method to check change of Custom_Status__c field to 'Reset'.(In Before trigger)
+	 */ 
+    @isTest
+    private static void testCustomStatusFieldChangedFromNewToReset()
+    {
+        Class__c oneClass = new Class__c(Name = 'Test Class', Custom_Status__c = 'New');
+        oneClass.MaxSize__c = 100;
+        insert oneClass;
+        List<Student__c> studentList = new List<Student__c>();
+        for(Integer i = 0; i < 101 ; i++)
+        {
+            studentList.add(new Student__c(Name='Test ' + i, class__c = oneClass.id));
+        }
+        insert studentList;
+        oneClass.Custom_Status__c = 'Reset';
+        Database.update(oneClass);
+        System.assertEquals(0, [Select COUNT() FROM Student__c WHERE class__c =: oneClass.id], 
```

Comment:
soql keywords should be in uppercase
API names of SObjects and fields should be in proper case
check in other places too

Reply:
Done